inst/shiny/modules/tab_visuals.R

```diff
@@ -201,7 +215,20 @@ tab_visuals_server <- function(id, data, grouping_vars, res_nca) {
       updatePickerInput(
         session,
         "generalplot_analyte",
-        choices = param_choices_analyte
+        choices = param_choices_analyte,
+        selected = param_choices_analyte[1]
```

Comment:
nitpick(non-blocking): No need to specify selected argument, upon choices change this will automatically change to the first element.

```suggestion
        choices = param_choices_analyte
```